src/Nordea/Components/Label.elm

```diff
@@ -137,7 +138,7 @@ view attrs children (Label config) =
                 ]
                 attrs
                 ((legend
-                    [ css [ padding (rem 0), flexBasis (pct 100), Css.Global.children [ everything [ flexBasis (pct 100) ] ] ] ]
+                    [ css [ width (pct 100), padding (rem 0), flexBasis (pct 100), Css.Global.children [ everything [ flexBasis (pct 100) ] ] ] ]
```

Comment:
I thought the flexBasis would do the same, but maybe it is shrinking? 🤔
```suggestion
                    [ css [ flexShrink (num 0), padding (rem 0), flexBasis (pct 100), Css.Global.children [ everything [ flexBasis (pct 100) ] ] ] ]
```

Reply:
`flexShrink` doesnt seem to work

Reply:
all right, but does width work if we remove flexBasis? If so, I think we should remove it